examples/player/simid_player.js

```diff
@@ -55,6 +55,12 @@ class SimidPlayer {
      */
     this.adComplete_ = adComplete;
 
+    /**
+     * A boolean indicating whether type of creative
```

Comment:
I think you're missing a couple words here

Reply:
weird, must have gotten deleted. Fixed